getting-started/templates/systemlink-values.yaml

```diff
@@ -627,6 +627,17 @@ dataframeservice:
   ## individual rates configured here.
   ##
   rateLimits:
+    ## Configuration for the global rate limiter that applies to all requests
+    ##
+    global:
+      ## Enable the rate limiter
+      enabled: true
```

Comment:
Should we also include `enabled: true` for `queryMetadata` and `queryRowData` for consistency? Or remove it for `global`?